diff --git a/BUILD.bazel b/BUILD.bazel
index 1f548b9b..04f792c3 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -208,6 +208,7 @@ swift_syntax_library(
 swift_syntax_test(
     name = "SwiftRefactorTest",
     deps = [
+        ":SwiftIDEUtils",
         ":SwiftRefactor",
         ":_SwiftSyntaxTestSupport",
     ],
@@ -317,6 +318,7 @@ swift_syntax_library(
     name = "SwiftSyntaxMacros",
     deps = [
         ":SwiftDiagnostics",
+        ":SwiftIfConfig",
         ":SwiftParser",
         ":SwiftSyntax",
         ":SwiftSyntaxBuilder",
